docs/examples/network_profile/profiles.yaml

```diff
@@ -0,0 +1,98 @@
+        to:
+        - podSelector: {}
+      - # rule: allow internet access
+        to:
+        - ipBlock:
+            # As "the internet" is hard to define as a list of IP blocks, we define it as
+            # any address with exceptions.
+            cidr: 0.0.0.0/0 # any
+            except:
+            # exclude special address blocks used for non-public networks
+            # see https://en.wikipedia.org/wiki/Reserved_IP_addresses
+            - 10.0.0.0/8 # address block for private networks
+            - 100.64.0.0/10 # shared address space for carrier-grade NAT
+            - 169.254.0.0/16 # address block for local links; used for metadata.google.internal
+            - 172.16.0.0/12 # address block for private networks; used for K8s masters
+            - 192.168.0.0/16 # address block for private networks; used for nodes, pods, services
+            - 198.18.0.0/15 # address block for benchmark testing in private networks
+      - # rule: allow access to cluster DNS
+        to:
+        - namespaceSelector: {}
+            # The namespace is 'kube-system', but has no labels
+            # attached. Therefore we cannot select it here.
+            # If we can set labels on the namespace, make this selector more
+            # specific.
+          podSelector:
+            matchExpressions:
+            - key: k8s-app
+              operator: In
+              values:
+              - kube-dns
+              - coredns
+        ports:
+        - protocol: UDP
+          port: 53
+        - protocol: TCP
+          port: 53
+        - port: 8053
+          protocol: UDP
+        - port: 8053
+          protocol: TCP
+      - # rule: allow access to K8s apiserver (cluster-internal address)
+        to:
+        - ipBlock:
+            cidr: 100.64.0.1/32
+        ports:
+        - port: 443
+      - # rule: allow access to Elasticsearch reverse proxy
+        to:
+        - namespaceSelector: {}
+            # The namespace is 'steward-elasticsearch-proxy', but has no labels
+            # attached. Therefore we cannot select it here.
+            # If we can set labels on the namespace, make this selector more
+            # specific.
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/name: steward-elasticsearch-proxy
+      - # rule: allow access to malware scan proxy
+        to:
+        - namespaceSelector: {}
+            # The namespace is 'malware-scan-proxy', but has no labels
+            # attached. Therefore we cannot select it here.
+            # If we can set labels on the namespace, make this selector more
+            # specific.
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/instance: malware-scan-proxy
+      ingress:
+      - # rule: allow access from all pods in the same namespace
+        from:
+        - podSelector: {}
+kind: ConfigMap
+metadata:
+  labels:
+  name: steward-pipelineruns-network-policies
+  namespace: steward-system
```

Comment:
The Steward system namespace might be different, so we should not include it here. Instead, it should be specified via kubectl argument.

Reply:
Removed